packages/wallets/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -20,40 +32,79 @@ const AddedMT5AccountsList: React.FC<TProps> = ({ account }) => {
     const { data: activeWallet } = useAuthorize();
     const { getVerificationStatus } = useJurisdictionStatus();
     const jurisdictionStatus = useMemo(
-        () => getVerificationStatus(account.landing_company_short || 'svg', account.status),
+        () => getVerificationStatus(account.landing_company_short || JURISDICTION.SVG, account.status),
         [account.landing_company_short, account.status, getVerificationStatus]
     );
-    const { title } = MarketTypeDetails[account.market_type ?? 'all'];
+    const { title } = MarketTypeDetails[account.market_type ?? MARKET_TYPE.ALL];
     const { isMobile } = useDevice();
     const { show } = useModal();
     const { t } = useTranslation();
 
+    const { getPlatformStatus } = useTradingPlatformStatus();
+    const platformStatus = getPlatformStatus(account.platform);
+
+    const hasPlatformStatus =
+        account.status === TRADING_PLATFORM_STATUS.UNAVAILABLE ||
+        platformStatus === TRADING_PLATFORM_STATUS.MAINTENANCE;
+
+    const getBadgeText = () => {
+        if (account.status === TRADING_PLATFORM_STATUS.UNAVAILABLE) return 'Account unavailable';
+        if (platformStatus === TRADING_PLATFORM_STATUS.MAINTENANCE) return 'Server maintenance';
+        return '';
+    };
```

Comment:
```suggestion
    const getBadgeText = () => {
        if (account.status === TRADING_PLATFORM_STATUS.UNAVAILABLE) return 'Account unavailable';
        if (platformStatus === TRADING_PLATFORM_STATUS.MAINTENANCE) return 'Server maintenance';
        return '';
    };
```
Best to move this check as well as `hasPlatformStatus` as a component `TradingStatusBadge` so that anything related with the check can be done there. Doing it like this could cause edge cases - example a badge with an empty string? In the case where nothing matches if u handle it from a component u can return null and dont render anything